instructure/canvas-ios
# Show attachments in message details

refs: MBL-13774
affects: Parent
release note: none


Test Plan
--------------------
1. go to message details with some attachments (twilson has many)
2. make sure attachments show when they should on the cell and don't show when they shouldn't
3. click an attachment and make sure it is shown in modal.
(THERE IS A KNOWN BUG IN VIEWING SOME OF THESE ATTACHMENTS IN new FileViewController, the second time it is viewed, it is not centered.  This can be addressed in separate ticket)
4. Attachments should also scroll properly if there are more attachments than width of screen.
